core/notifications/src/graphql/schema.rs

```diff
@@ -13,50 +13,149 @@ pub struct Query;
 #[Object]
 impl Query {
     #[graphql(entity)]
-    async fn consumer_account(&self, id: ID) -> Option<ConsumerAccount> {
```

Comment:
For backwards compatibility won't we need to keep it on consumer_account as well?

Can still return the same
```
app
  .notification_settings_for_user(GaloyUserId::from(self.id.0.clone()))
  .await?;
```

Reply:
The types are differnent now and I think we will need to keep backwards compatibility by keeping it live on core for a period.

Reply:
In addition to the mutations on core.

Reply:
So core will proxy to notifications I for a while I guess.